Adjust includes in various places.

level_zero/tools/source/metrics/linux/os_metric_enumeration_imp_linux.cpp:14:10: fatal error: 'sys/sysmacros.h' file not found
#include <sys/sysmacros.h>
         ^~~~~~~~~~~~~~~~~
In file included from level_zero/tools/source/sysman/linux/os_sysman_imp.h:12:
level_zero/tools/source/sysman/linux/fs_access.h:19:10: fatal error: 'sys/sysmacros.h' file not found
#include <sys/sysmacros.h>
         ^~~~~~~~~~~~~~~~~

core/os_interface/linux/drm_neo.cpp:21:10: fatal error: 'linux/limits.h' file not found
#include <linux/limits.h>
         ^~~~~~~~~~~~~~~~

runtime/tbx/tbx_sockets_imp.cpp:74:51: error: use of undeclared identifier 'IPPROTO_TCP'
        m_socket = ::socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
                                                  ^
runtime/tbx/tbx_sockets_imp.cpp:107:9: error: unknown type name 'sockaddr_in'; did you mean 'sockaddr'?
        sockaddr_in clientService;
        ^~~~~~~~~~~
        sockaddr
/usr/include/sys/socket.h:328:8: note: 'sockaddr' declared here
struct sockaddr {
       ^

--- level_zero/tools/source/metrics/linux/os_metric_oa_enumeration_imp_linux.cpp.orig	2021-01-20 18:22:07 UTC
+++ level_zero/tools/source/metrics/linux/os_metric_oa_enumeration_imp_linux.cpp
@@ -13,7 +13,9 @@
 #include "level_zero/tools/source/metrics/metric_oa_source.h"
 
 #include <sys/stat.h>
+#if defined(__linux__)
 #include <sys/sysmacros.h>
+#endif
 
 namespace L0 {
 
--- level_zero/tools/source/sysman/linux/fs_access.h.orig	2020-03-27 09:53:58 UTC
+++ level_zero/tools/source/sysman/linux/fs_access.h
@@ -16,7 +16,6 @@
 #include <sstream>
 #include <string>
 #include <sys/stat.h>
-#include <sys/sysmacros.h>
 #include <sys/types.h>
 #include <unistd.h>
 #include <vector>
--- shared/source/os_interface/linux/drm_neo.cpp.orig	2020-04-17 18:43:38 UTC
+++ shared/source/os_interface/linux/drm_neo.cpp
@@ -19,9 +19,9 @@
 #include "shared/source/os_interface/os_interface.h"
 #include "shared/source/utilities/directory.h"
 
+#include <climits>
 #include <cstdio>
 #include <cstring>
-#include <linux/limits.h>
 
 namespace NEO {
 
--- shared/source/tbx/tbx_sockets_imp.cpp.orig	2020-10-23 15:44:19 UTC
+++ shared/source/tbx/tbx_sockets_imp.cpp
@@ -18,6 +18,7 @@ typedef int socklen_t;
 #else
 #include <arpa/inet.h>
 #include <netdb.h>
+#include <netinet/in.h>
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
